### DISCUSSION TITLE
Unsafe Files???

### DISCUSSION THREAD

**Discussion by gohan2091** [Bing-su/adetailer#470] [Q&A]
What's up with Huggingface saying 5 Adetailer files are marked as unsafe?: https://huggingface.co/Bingsu/adetailer/tree/main Can anyone explain? I wanted to try the m models out and the fashion one but will hold off and wanted to check here. Anyone know?

**Comment by Bing-su** [comment]
https://huggingface.co/docs/hub/security-pickle

The file uploaded to huggingface was created by me.

**Comment by gohan2091** [comment]
Does that answer my question though? Is it saying all pickle files are marked as security issues?

**Reply by brknsoul** [reply]
pickle files can run arbitrary code. It doesn't mean that it actually does do this.
Like any file you download from the internet, ensure you trust where and who the file's coming from.

**Reply by gohan2091** [reply]
So basically I should trust these files because they are made by the Adetailer guy above and he is to be trusted? I'm wondering why there are no safetensor versions also.

**Reply by brknsoul** [reply]
I think it's because adetailer hasn't been coded to accept safetensor files.
You might remove the tinfoil hat. ;-)
